### PR TITLE
Csa 0000 home screen content

### LINKED ISSUE #25: Feature - Retrieve Contact data on the Landing Screen (Home Screen)
On Home Screen, all the contact details from database should be loaded in the Card designed.

### PR DESCRIPTION
**Pull request by AmanKamani**
closes #25

### CODE REVIEW

File: app/src/main/java/com/jb/svb/contactsaver/repositories/ContactRepository.kt [hunk 1]
```diff
@@ -5,11 +5,17 @@ import com.jb.svb.contactsaver.models.Contact
 import com.jb.svb.contactsaver.models.asDatabaseModel
 import com.jb.svb.contactsaver.persistance.dao.ContactDao
 import com.jb.svb.contactsaver.persistance.entities.ContactEntity
+import com.jb.svb.contactsaver.persistance.entities.asModel
+import kotlinx.coroutines.flow.map
 
 class ContactRepository(
     private val contactDao: ContactDao
 ) {
```
Comment (shivaprasad-bhat) [inline app/src/main/java/com/jb/svb/contactsaver/repositories/ContactRepository.kt:14]: Can you remove un-necessary space

**Review by shivaprasad-bhat — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/jb/svb/contactsaver/repositories/ContactRepository.kt (hunk 1 above)
Comment (AmanKamani) [reply app/src/main/java/com/jb/svb/contactsaver/repositories/ContactRepository.kt:14]: Fixed

Comment (shivaprasad-bhat) [reply app/src/main/java/com/jb/svb/contactsaver/repositories/ContactRepository.kt:14]: Good to go

**Review by shivaprasad-bhat — APPROVED** [review]

**Review by shivaprasad-bhat — APPROVED** [review]

**Comment by shivaprasad-bhat** [comment]
Check the CircleCI issue and address it

**Comment by shivaprasad-bhat** [comment]
/home/circleci/project/app/src/main/java/com/jb/svb/contactsaver/core/LiveDataValidator.kt:24: Error: Cannot set non-nullable LiveData value to null [NullSafeMutableLiveData]
          errorLatest.value = null

**Review by shivaprasad-bhat — APPROVED** [review]

**Comment by AmanKamani** [comment]
> Check the CircleCI issue and address it

Fixed. @shivaprasad-bhat

**Pull request merged**